tutorials/commands/flux-mini-submit.rst

```diff
@@ -0,0 +1,74 @@
+.. _flux-mini-submit:
+
+==========================
+How to Submit Jobs in Flux
+==========================
+
+Do you want to submit a job to Flux? Here's a short tutorial on how to do so!
+
+----------------------------
+Submit Jobs Using Flux's CLI
+----------------------------
+
+From within a Flux instance, you can submit your job on the command line with
+optional arguments, resource options, per task options, and per resource
+options:
+
+.. code-block:: sh
+
+    $ flux mini submit --nodes=2 --ntasks=4 --cores-per-task=2 ./my_compute_script.lua 120
```

Comment:
minor nit, should we do `.py` everywhere instead of `.lua`?  since python is more popular.

Reply:
Oh, sure! Good call. Just force-pushed a fix to use `.py` instead of `.lua`